### ISSUE TITLE
intelx API not working

### ISSUE THREAD

**Issue by motoko-ayanami** [projectdiscovery/subfinder#919]
Version v2.6.1

API key for intelx.io doesn't return any results. It isn't clear how we are supposed to insert the API key for this site.
![intelx](https://github.com/projectdiscovery/subfinder/assets/122310140/95d84d10-4b03-4314-8d5e-a511a0f71a8b)

**Comment by MetzinAround** [comment]
Hi! 

Just checking, do you have the API key in your provider-config.yaml file?

**Comment by motoko-ayanami** [comment]
Yup, api key is there, but maybe I need to add something extra using the colon?

**Comment by MetzinAround** [comment]
I remember having a hard time with the formatting of my keys list before. Double check it against the one in the readme: https://github.com/projectdiscovery/subfinder#post-installation-instructions

I had it a little goofy before I checked that. 

HOWEVER, I will say this is the second person I've heard an issue with intelx or some other API not working. A likely culprit might be some APIs updating their endpoints.

**Comment by rampwut** [comment]
Dear Team.

It doesn't work for me as well.. :(

![image](https://github.com/projectdiscovery/subfinder/assets/141624170/e70bb592-f3b2-4716-9447-c1720c5c4c8f)

i defined api key inside this config file: ` /root/.config/subfinder/provider-config.yaml`

**Comment by ehsandeep** [comment]
@motoko-ayanami @rampwut I've added an example input of api key here https://github.com/projectdiscovery/subfinder/pull/923, you need to host as well followed by api key. 

```yaml
intelx:
  - 2.intelx.io:s4324-b98b-41b2-220e8-3320f6a1284d
```
